src/components/NotificationModal.tsx

```diff
@@ -141,26 +127,38 @@ export const NotificationModal: React.FC<ModalProps> = ({
             </Option>
           ))}
         </StyledSelect>
-      </>
-      {extraFields &&
-        extraFields.map(({ text, key }) => (
-          <ExtraFieldContainer key={key}>
-            <TextInput
-              label=" "
-              data-cy={`${key}-input`}
-              description={text}
-              onChange={(event) => {
-                setExtraFieldInputVals({
-                  ...extraFieldInputVals,
-                  [key]: event.target.value,
-                });
-              }}
-              value={extraFieldInputVals[key]}
-            />
-          </ExtraFieldContainer>
-        ))}
+      </Section>
+      {extraFields && extraFields.length && (
+        <Section>
+          <div>
+            <Body weight="medium">Choose how to be Notified</Body>
+          </div>
+          {extraFields.map(({ text, key }) => (
+            <ExtraFieldContainer key={key}>
+              <LabelContainer>
+                <InputLabel>{text}</InputLabel>
+              </LabelContainer>
+              <StyledInput
+                data-cy={`${key}-input`}
+                onChange={(event) => {
+                  setExtraFieldInputVals({
+                    ...extraFieldInputVals,
+                    [key]: event.target.value,
+                  });
+                }}
+                value={extraFieldInputVals[key]}
+              />
+            </ExtraFieldContainer>
+          ))}
+        </Section>
+      )}
       <div>
-        then notify by:
+        <div>
+          <Body weight="medium">Choose how to be Notified</Body>
```

Comment:
Changing the label here makes the options in the select have the wrong tense. Can you remove -ing from the option labels?
<img width="650" alt="Screen Shot 2020-06-18 at 9 14 18 AM" src="https://user-images.githubusercontent.com/29384396/85024666-65adc180-b144-11ea-8055-fe90ca1518a3.png">